crates/egui/src/widgets/plot/mod.rs

```diff
@@ -73,36 +73,25 @@ impl Default for CoordinatesFormatter {
 const MIN_LINE_SPACING_IN_POINTS: f64 = 6.0; // TODO(emilk): large enough for a wide label
 
 #[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
-#[derive(Clone)]
-struct AutoBounds {
+#[derive(Copy, Clone)]
+struct AxisBools {
     x: bool,
     y: bool,
 }
 
-impl AutoBounds {
-    fn from_bool(val: bool) -> Self {
-        AutoBounds { x: val, y: val }
-    }
-
-    fn any(&self) -> bool {
-        self.x || self.y
-    }
-}
-
-impl From<bool> for AutoBounds {
+impl From<bool> for AxisBools {
     fn from(val: bool) -> Self {
-        AutoBounds::from_bool(val)
+        AxisBools { x: val, y: val }
     }
 }
 
 /// Information about the plot that has to persist between frames.
 #[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
 #[derive(Clone)]
 struct PlotMemory {
-    auto_bounds: AutoBounds,
+    modified: AxisBools,
```

Comment:
Has _what_ been modified? Bounds? If so `bounds_modified` is a better name.

Reply:
Yeah. Changed the name and added some documentation to the field.